vgscollect-cardio/src/main/java/com/verygoodsecurity/api/cardio/ScanActivity.kt

```diff
@@ -58,10 +58,10 @@ class ScanActivity : BaseTransmitActivity() {
             .putExtra(CardIOActivity.EXTRA_KEEP_APPLICATION_THEME, keepApplicationTheme)
 
 
-        suppressManualEnter?.let {
+        suppressManualEnter.let {
             scanIntent.putExtra(CardIOActivity.EXTRA_SUPPRESS_MANUAL_ENTRY, it)
         }
-        suppressConfirmation?.let {
+        suppressConfirmation.let {
```

Comment:
@DonaldRG If `suppressConfirmation` is not nullable we can remove `let`.

Reply:
Yup, that make sense, and same for `suppressManualEnter`